db/schema.rb

```diff
@@ -62,10 +62,10 @@
     t.integer  "gift_request_id"
     t.integer  "donor_id"
     t.string   "tracking_num"
-    t.boolean  "delinquent"
+    t.boolean  "delinquent",      default: false
     t.string   "state"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+    t.datetime "created_at",                      null: false
```

Comment:
`schema.rb` noise is a classic problem with Rails development - basically it put it in to put `null` in a different column from `default`. Nothing for it, unfortunately.